package.json

```diff
@@ -57,7 +57,7 @@
     "@sewing-kit/plugins": "^0.1.10",
     "@shopify/app-bridge": "^0.7.3",
     "@shopify/babel-preset": "^23.1.1",
-    "@shopify/eslint-plugin": "^36.0.2",
+    "@shopify/eslint-plugin": "^39.0.0",
```

Comment:
Could this be split out into a separate PR? I'm finding it difficult to split out the graphql changes from the lint changes.

Reply:
it is now https://github.com/Shopify/quilt/pull/1880/checks?check_run_id=2504490247